app/src/main/java/org/mozilla/reference/browser/settings/CustomColorPreference.kt

```diff
@@ -0,0 +1,49 @@
+package org.mozilla.reference.browser.settings
+
+import android.content.Context
+import android.support.v4.content.ContextCompat
+import android.support.v7.preference.Preference
+import android.support.v7.preference.PreferenceViewHolder
+import android.util.AttributeSet
+import android.widget.TextView
+import org.mozilla.reference.browser.R
+
+/**
+ * This preference is used to define custom  colors for both title and summary texts.
+ * Color code #777777 (placeholder_grey) is used as the fallback color for both title and summary.
+ */
+class CustomColorPreference : Preference {
+    private var mTitleColor: Int = 0
```

Comment:
Nit: I believe [Hungarian Notation](https://jakewharton.com/just-say-no-to-hungarian-notation/) is disuse pattern.